lib/datadog/profiling/tag_builder.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
 
 require_relative '../core/utils'
+require_relative '../core/environment/git'
+require_relative '../core/git/ext'
 
 module Datadog
   module Profiling
     # Builds a hash of default plus user tags to be included in a profile
     module TagBuilder
```

Comment:
@ivoanjo could you review these changes? I tried to follow the conventions I saw in this file but I am not sure if I'm doing it correctly

Reply:
See https://github.com/DataDog/dd-trace-rb/pull/3424/#discussion_r1491157422 :)